### DISCUSSION TITLE
Moving library to new computer

### DISCUSSION THREAD

**Discussion by ghost** [audiamus/BookLibConnect#160] [Q&A]
I’ve moved my audiobook library to a new computer I copied over the folder from the old computer and installed Book Lib Connect on the new computer, I set the download folder as the one I copied but it’s not recognizing the books I’ve already downloaded. Is that just not possible or is there a way for it to recognize the books I already have?

**Comment by audiamus** [comment]
You can copy everything under `C:\Users\<user>\AppData\Local\audiamus\BookLibConnect`. (You'll need `config`, `data`, `img `and `settings`.) However, `config\configuration.json` must be decrypted first because it is bound to your computer hardware for better security. You can decrypt and encrypt in the Profile dialog, Don't forget to encrypt again on the new computer.
